docs/sql/float.md

```diff
@@ -27,7 +27,7 @@ an example is `NaN` raised to the zero power yields one.
 In sorting order `NaN` is considered greater than all other values.
 
 The legal operations are `+` (plus, unary and binary), `-` (minus,
-unary and binary), `*` (multiplication), `/` (division).
+unary and binary), `*` (multiplication), `/` (division), `%`
```

Comment:
Can you find somewhere a description of what % does?
Remember that for integers you had to describe the 4 cases based on sign.

Reply:
you are missing the documentation for decimal

Reply:
Postgres leaves it at this: 
```sql
numeric_type % numeric_type → numeric_type
-- Modulo (remainder); available for smallint, integer, bigint, and numeric

-- Example(s)
5 % 4 → 1
```